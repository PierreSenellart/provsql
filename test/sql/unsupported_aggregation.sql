\set ECHO none
\pset format unaligned
SET search_path TO provsql_test,provsql;

SELECT time, person, COUNT(room)
FROM s
GROUP BY time, person
HAVING COUNT(room) > 1 AND COUNT(room) < 5;
