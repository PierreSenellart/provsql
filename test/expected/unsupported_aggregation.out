\set ECHO none
ERROR:  ProvSQL cannot handle complex HAVING expressions
